### PR TITLE
Admissions copy review

### PR DESCRIPTION
**Pull request by cimendes**
This PR updates the templates for the admissions. 

**Work In Progress:** Add refund policy to code of conduct template. (waiting for merge in the wiki)

### CODE REVIEW

File: docker/local/django/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-slim-buster
+FROM python:3.8-slim-buster
```
Comment (dominguezmaria) [inline docker/local/django/Dockerfile:1]: should this be changed? 🤔 (just "atirating uma laracha" 😂 )

**Review by dominguezmaria — COMMENTED** [review]
So far so good! You rock @cimendes !!
Waiting on the refund policy stuff to approve.

File: docker/local/django/Dockerfile (hunk 1 above)
Comment (cimendes) [reply docker/local/django/Dockerfile:1]: OPS! my bad, blame the M1...

File: portal/templates/candidate_templates/home.html [hunk 2]
```diff
@@ -10,51 +10,57 @@ <h1 class="py-5">Hello {% if first_name %}{{ first_name }}{% else %}future data
...
+        Thank you for your interest in participating in <b>Batch 5</b> of the LDSSA!<br>
+        Before you start, please read all the details about the LDSSA <a href="https://ldssa.github.io/wiki/Starters%20Academy%20(LDSSA)/01-Starters-Academy-(Course)/">here</a>, about this admissions process <a href="https://ldssa.github.io/wiki/Applicants/Application-process/">here</a>, and keep these main points in mind:
     </p>
 
     <ul>
-        <li class="pb-3">In order to get the most out of the experience, students should spend an average of <b>10h per week</b> (and no less than 5h per week) studying on their own time. The Bootcamp and Hackathons are not included in this time commitment.</li>
+        <li class="pb-3">
+            In order to get the most out of the experience, students should spend an average of <b>10 hours per week</b> (and no less than 5 hours per week) studying on their own time. The Bootcamp and Hackathons are not included in this time commitment.</li>
```
[the remark this answers is not in the document]
Comment (cimendes) [reply portal/templates/candidate_templates/home.html]: Added explicit time commitment for bootcamp and hackathons

File: portal/templates/candidate_templates/home.html [hunk 3]
```diff
@@ -10,51 +10,57 @@ <h1 class="py-5">Hello {% if first_name %}{{ first_name }}{% else %}future data
...
         <li class="pb-3">
             <b>The Starters Academy will be remote this year.</b> This means that, until further notice, all events are scheduled to happen remotely.
-            In case it becomes safe, we may decide to have the later hackathons in person, but students may still join them remotely.
+            In case it becomes safe, the later Hackathons might be held in person, but students will always have the option to join them remotely.
```
[the remark this answers is not in the document]
Comment (cimendes) [reply portal/templates/candidate_templates/home.html]: removed mention of in person events

**Comment by cimendes** [comment]
@dominguezmaria refund policy has now been added

**Review by hcastilho — APPROVED** [review]

**Pull request merged**